src/lib/filterAtfs.ts

```diff
@@ -1,16 +1,16 @@
 import { utcToZonedTime } from 'date-fns-tz';
-import { isValid } from 'date-fns';
+import { isValid, parseISO } from 'date-fns';
 import { AuthorisedTestingFacility } from '../models/authorisedTestingFacility';
 
 // eslint-disable-next-line max-len
 const removeAtfsWithNoAvailability = (items: AuthorisedTestingFacility[]): AuthorisedTestingFacility[] => items.filter((item) => item?.availability?.isAvailable !== false
-        || !isAvailabilityInformationUpToDate(item?.availability?.endDate));
+        || !isAvailabilityInformationUpToDate(parseISO(item?.availability?.endDate)));
 
-const isAvailabilityInformationUpToDate = (date: string | undefined): boolean => date
-    && isValid(date) && !isDateBeforeToday(date);
+// eslint-disable-next-line max-len
```

Comment:
Is the eslint max-len too restrictive? We need to change the lint value or abide by them

Reply:
I think so yeah. It also kind of conflicts with another rule that you can't immediately take a new line after the arrow syntax when not using brackets. And then if you do use brackets and take a new line, it complains about another rule that you shouldn't be using brackets for one liner returns. We can maybe discuss which ones we want to keep and which ones to get rid of